### DISCUSSION TITLE
Is it possible to implement such a function: set a certain area of ​​a nested node as the range of its child node?

### DISCUSSION THREAD

**Discussion by egglemmon** [bcakmakoglu/vue-flow#1688] [Q&A] [answered]
Set a certain area of ​​the nested node as the range of its child node. The child node can extend the parent node (implemented), and the child node has a margin from the parent node.

https://github.com/user-attachments/assets/337a27be-a1e6-47e9-b2d6-8935f505a506

**Comment by bcakmakoglu** [comment, marked as answer]
It is possible to use a padding for extents but you can't couple a parent extent *and* paddings (since that just doesn't really make sense).

If you wanna use a padding something like this is possible

```ts
extent: {
	range: 'parent',
	padding: 10,
}

**Reply by egglemmon** [reply]
Thanks for the timely reply. What should I do if I want to combine extent and paddings in the current API? can it be done ?

**Reply by egglemmon** [reply]
Can we finally achieve the effect and interaction of video?

**Reply by bcakmakoglu** [reply]
> Thanks for the timely reply. What should I do if I want to combine extent and paddings in the current API? can it be done ?

I gave an example above already or what are you talking about? ^^

**Comment by egglemmon** [comment]
https://reactflow.dev/examples/nodes/dynamic-grouping

Is it possible to achieve the effect of this dynamic group?

**Reply by bcakmakoglu** [reply]
Yeah it is but since this is a Pro example I will *not* provide any code for this whatsoever.
You'll have to figure it out yourself.

**Reply by egglemmon** [reply]
I understand, thank you very much
